src/data-model/item-data/skillData.ts

```diff
@@ -46,6 +46,9 @@ export interface SkillDataProperties {
 }
 
 export const emptySkill: SkillDataSourceData = {
+  rqid: "",
+  rqidpriority: 0,
+  rqidlang: "",
```

Comment:
Made these constants in IRqid.ts and used them in the other classes so we can change them in one place. Thought about making them settings but that wouldn't work because these get baked into the template.json at compile time.